orangecontrib/educational/widgets/owkmeans.py

```diff
@@ -393,36 +374,39 @@ def complete_replot(self):
 
         # plot centroids
         options = dict(series=[])
-        options['series'].append(dict(data=[{'x': p[0],
-                                             'y': p[1],
-                                             'marker':{'fillColor': self.colors[i % len(self.colors)]}}
-                                            for i, p in enumerate(self.k_means.centroids)],
-                                      type="scatter",
-                                      draggableX=True if self.k_means.step_completed else False,
```

Comment:
Have I overlooked something here or is `self.k_means.step_completed` already `bool`? If not `bool(self.k_means.step_completed)` would have the same effect as this line.